### PR TITLE
IDE-985 Support locale specific help file

### PR DESCRIPTION
**Pull request by GordonSmith**
Specifically pt-BR

Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@richardkchapman this should really target master...
@dehilsterlexis please review

File: wlib/PreferenceDlg.cpp [hunk 1]
```diff
@@ -1964,6 +1951,15 @@ class CPrefOtherDlg : public CDialogImpl<CPrefOtherDlg>,
 		DoChanged(false);
 	}
 
+	void InitComboHelpLocale()
+	{
+		m_comboHelpLocale = GetDlgItem(IDC_COMBO_HELP);
+		m_comboHelpLocale.AddString(_T(""));
+		m_comboHelpLocale.AddString(_T("en"));
```
Comment (dehilsterlexis) [inline wlib/PreferenceDlg.cpp:1958]: All uppercase "en" and "pt-BR"

Comment (GordonSmith) [reply wlib/PreferenceDlg.cpp:1958]: No - I am keeping them the same casing as what the OS reports when auto detecting.

**Comment by dehilsterlexis** [comment]
@GordonSmith @richardkchapman tested and looks good

**Pull request merged**